src/main/java/jenkins/plugins/logstash/LogstashWriter.java

```diff
@@ -149,10 +148,8 @@ public boolean isConnectionBroken() {
   }
 
   // Method to encapsulate calls for unit-testing
-  LogstashIndexerDao createDao() throws InstantiationException {
-    LogstashInstallation.Descriptor descriptor = LogstashInstallation.getLogstashDescriptor();
-    return IndexerDaoFactory.getInstance(descriptor.getType(), descriptor.getHost(), descriptor.getPort(),
-        descriptor.getKey(), descriptor.getUsername(), descriptor.getPassword());
+  LogstashIndexerDao getIndexerDao() {
```

Comment:
I don't think this method is called anymore.
Inline?

Reply:
correct will make it private

Reply:
yeah, sorry my IDE somehow missed that usage